zebra-chain/src/block/serialize.rs

```diff
@@ -115,6 +121,12 @@ impl ZcashSerialize for Block {
 
 impl ZcashDeserialize for Block {
     fn zcash_deserialize<R: io::Read>(reader: R) -> Result<Self, SerializationError> {
+        // # Consensus
+        //
+        // > The size of a block MUST be less than or equal to 2000000 bytes.
+        //
+        // https://zips.z.cash/protocol/protocol.pdf#blockheader
```

Comment:
I wonder if we should also document the consensus rule on the `MAX_BLOCK_BYTES` constant.

What do you think?

Reply:
I will prefer to avoid quoting the same consensus rule twice specially if they are in the same file as in this case. I think we should document the consensus rule just where is checked/applied when this is possible.